auto_submit/lib/request_handling/pubsub.dart

```diff
@@ -20,6 +20,7 @@ class PubSub {
 
   final HttpClientProvider httpClientProvider;
 
+  /// Adds one or more messages to the topic.
```

Comment:
In it's current form, this only pushes one message

Reply:
Changed it to 'Adds one message to the topic.'